angularjs-portal-home/src/main/webapp/my-app/layout/partials/home-header.html

```diff
@@ -3,8 +3,8 @@
     <md-switch class="md-accent md-hue-2 app-header-toggle" ng-model="expandedMode"
                aria-label="Toggle expanded widget layout" ng-change="toggleMode(expandedMode)"
                style="margin-top:0;margin-bottom:0;">
-      <span hide-sm hide-xs>Show expanded mode</span>
-      <span hide-gt-sm>Show Expanded</span>
+      <span hide-sm hide-xs>Expand widgets</span>
+      <span hide-gt-sm>Expand Widgets</span>
```

Comment:
@apetro: It's written both ways at the moment because everything in md-menus is capitalized. I'm going to write a PR for frame that makes items in md-menus sentence-case as well (that would be consistent with google's guidelines).